.github/workflows/release-notifier.yml

```diff
@@ -0,0 +1,33 @@
+name: Create Issue On Release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  create-issue:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Create Issue
+      uses: actions/github-script@v6
+      with:
+        github-token: ${{ secrets.DOCS_PAT }}  # Assuming 'PAT' is your Personal Access Token stored as a secret
```

Comment:
whose PAT is this? do we have a way to manage them? have run into issues where they expire and we're not sure whose they were

Reply:
this one won't expire. If it fails then can remove this action, or update it with a new one any time. Hopefully in future a github token will do the trick but haven't worked that out yet.